python/lsst/afw/cameraGeom/camera.py

```diff
@@ -65,8 +65,31 @@ def _tranformFromNativeSys(self, nativePoint, toSys):
         else:
             return self._transformSingleSys(nativePoint, toSys)
 
+    def _transformSingleSysArray(self, positionArray, fromSys, toSys):
+        """!Transform an array of points from once CameraSys to another CameraSys
+        @warning This method only handles a single jump, not a transform linked by a common native sys.
+        
+        @param[in] positionArray Array of Point2D objects, one per position
+        @param[in] fromSys  Initial coordinate system
+        @param[in] toSys  Destination coordinate system
```

Comment:
On the whole this method looks fine. Two requested tweaks:
- Rename variable positionArray to positionList, our generic name for an ordered collection
- For the parameter documentation change "coordinate system" to "camera coordinate system"